tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -2354,7 +2355,8 @@ public static function getActivities(
 		string $user,
 		string $password,
 		string $resourceId,
-		?array $filterParams = []
+		?array $filterParams = [],
+		?string $sortValue = null
```

Comment:
You should make use of `$filterParams`. it should handle all filter options
```suggestion
		?array $filterParams = [],
```

Reply:
Alright. :+1: